.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
might be good to also include repro steps - especially if there is a script that can reproduce the issue.  Getting the dataset and a script to repro the problem is usually the first step when trying to debug issues.

Also might be good to have a section on system information (windows, mac, linux, etc).

Reply:
Good points. Updated to include that and more. thanks!